packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -30,17 +31,22 @@ const SettingsSidebar = ( { sidebarName } ) => (
 		<SettingsHeader sidebarName={ sidebarName } />
 		<Panel>
 			{ sidebarName === 'edit-post/document' && (
-				<Fragment>
-					<PostStatus />
-					<LastRevision />
-					<PostLink />
-					<PostTaxonomies />
-					<FeaturedImage />
-					<PostExcerpt />
-					<DiscussionPanel />
-					<PageAttributes />
-					<MetaBoxes location="side" />
-				</Fragment>
+				<PluginDocumentSetting.Slot>
```

Comment:
What was the reason for using render function which wraps all existing panels? It can be also defined in place of `{ fills }` as `<PluginDocumentSetting.Slot />`.

Reply:
This was modeled after the approach taken for the `<PostStatus />` component. https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/components/sidebar/post-status/index.js#L29

If this is incorrect, I'm happy to change it but I think we should standardize whatever approach is taken.

Out of curiosity, can you demonstrate what that looks like and explain any differences/benefits to the other approach?